test/sdc-api.spec.js

```diff
@@ -9,6 +9,8 @@ const app = require('../server/server');
 
 const request = supertest(app);
 
+beforeAll(() => jest.setTimeout(15 * 1000));
```

Comment:
what's the reason for this?

Reply:
My tests were running before supertest had time to load up the server,etc. so the first GET test was always failing. I needed to add a setTimeout to avoid that.